src/components/PlantingSites/index.tsx

```diff
@@ -1,6 +1,10 @@
 import { Box } from '@mui/material';
-import strings from 'src/strings';
+import EmptyStatePage from '../emptyStatePages/EmptyStatePage';
 
 export default function PlantingSitesList(): JSX.Element {
-  return <Box>{strings.PLANTING_SITES}</Box>;
+  return (
+    <Box>
```

Comment:
i think we may need to add the same components like in seedbanks / nurseries etc.. there's some extra padding in the planting sites view.
fwiw, i think the extra padding in planting sites view looks cleaner - it helps fix https://terraformation.atlassian.net/browse/SW-2187 but not sure what the spec is.

Reply:
Mmm but seedbanks and nurseries have the emptyComponent called directly from the app.tsx. I could add the same components as inventory that is the only one calling the empty state inside the component